### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by TahenyBELGUITH**
In this project 
1️⃣  Added the tribunal home page
2️⃣ added js

### CODE REVIEW

**Comment by TahenyBELGUITH** [comment]
Hi @TahenyBELGUITH,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Required Changes ♻️
Check the comments under the review.



Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: tribute.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+body {
```
Comment (TahenyBELGUITH) [inline tribute.css:1]: kindly use the box sizing

File: tribute.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            <p>1983 - helps seven African countries dramatically increase their maize and sorghum yields</p>
+            <p>1984 - becomes a distinguished professor at Texas A&M University</p>
+            <p>2005 - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</p>
+            <p>2009 - dies at the age of 95.</p><br><br>
+            <p>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</p>
+            <br>
+            <p>-- Indian Prime Minister Manmohan Singh</p><br>
+
+            <p>If you have time, you should read more about this incredible human being on his <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank" id="tribute-link">Wikipedia entry.</a></p>
```
Comment (TahenyBELGUITH) [inline tribute.html:43]: it's rather to use just one p tag and spans

**Review by TahenyBELGUITH — COMMENTED** [review]
Hi @TahenyBELGUITH,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Required Changes ♻️
1️⃣ check your linters it's mandatory 
2️⃣ Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @TahenyBELGUITH , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :1st_place_medal:
Congratulations! 🎉 

:heavy_check_mark:  You have done well with all the project requirements.

### Optional Suggestions
 

- [ ] Check the comments under the review. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by AkashaRojee** [comment]
Hi @TahenyBELGUITH 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request that has already been reviewed and closed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AkashaRojee — COMMENTED** [review]
Hi @TahenyBELGUITH 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a pull request that has already been reviewed and closed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**